Makefile

```diff
@@ -132,7 +132,7 @@ release-ios: watchman-clean ##@build build release for iOS release
 	$(MAKE) jsbundle-ios && \
 	echo "Build in XCode, see https://status.im/build_status/ for instructions" && \
 	scripts/copy-translations.sh && \
-	open ios/StatusIm.xcworkspace
+	xcodebuild -workspace ios/StatusIm.xcworkspace -scheme StatusIm -configuration Release -destination 'generic/platform=iOS' -UseModernBuildSystem=N clean archive
```

Comment:
Just wondering (must have missed this earlier): why do we disable XCode's modern build system?

Reply:
This has been there for quite a while, so I don't remember the details, but I believe there was a build error that wasn't easy to address without it.